README.md

```diff
@@ -2,7 +2,28 @@
 
 [![License](https://licensebuttons.net/l/by-nc-nd/3.0/88x31.png)](https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode)
 
+ToolkitICL is an open source tool useful for automated execution  
```

Comment:
I would delete that sentence and add it as description at the top of this project (with a "." at the end).

Reply:
Indeed, this sentence is redundant and seems to be a remnant of the past...